code/modules/organs/subtypes/machine.dm

```diff
@@ -62,26 +62,101 @@
 	desc = "A small, powerful cell for use in fully prosthetic bodies."
 	icon = 'icons/obj/power.dmi'
 	icon_state = "scell"
-	organ_tag = "cell"
+	organ_tag = BP_CELL
 	parent_organ = BP_CHEST
-	vital = 1
-	var/emp_counter = 0
+	vital = TRUE
+	max_damage = 80
+	relative_size = 80
+	var/open = FALSE
+	var/obj/item/cell/cell = /obj/item/cell/super
+	//at 0.8 completely depleted after 60ish minutes of constant walking or 130 minutes of standing still
+	var/servo_cost = 0.8
 
 /obj/item/organ/internal/cell/Initialize()
 	robotize()
+	if(ispath(cell))
+		cell = new cell(src)
```

Comment:
This shouldn't be allowed to fail silently if the defined cell is not a valid path.
```suggestion
	cell = new cell(src)
```